addons/mail/static/src/core/common/message_model.js

```diff
@@ -115,14 +58,24 @@ export class Message extends Record {
     reactions = Record.many("MessageReactions", { inverse: "message" });
     notifications = Record.many("Notification", { inverse: "message" });
     recipients = Record.many("Persona");
-    /** @type {number|string} */
-    res_id;
-    /** @type {string|undefined} */
-    model;
+    originThread = Record.one("Thread");
```

Comment:
It should be a compute too to have the correct value when receiving old api data (only `res_id` and `model`)?